uart/lcd_uart/README.adoc

```diff
@@ -0,0 +1,39 @@
+= Attaching a 16x2 LCD via TTL
+
+This example code shows how to interface the Raspberry Pi Pico to one of the very common 16x2 LCD character displays. Due to the large number of pins these displays use, they are commonly used with extra drivers or backpacks. In this example, we will use an Adafruit LCD display backpack, which supports communication over USB or TTL. A monochrome display with an RGB backlight is also used, but the backpack is compatible with monochrome backlight displays too. There is <<_attaching_a_16x2_lcd_via_i2c, another example>> that uses I2C to control a 16x2 display.
```

Comment:
@JamesH65 There's currently no way for a pico-examples `README.adoc` to link to somewhere else in the databook, which means that this link won't work. So `<<_attaching_a_16x2_lcd_via_i2c, another example>>` should be changed to just `another example` .